docs/.vitepress/theme/Layout.vue

```diff
@@ -1,5 +1,19 @@
 <template>
 	<Layout>
+		<template #doc-before>
+			<div
+				v-if="path.startsWith('/packages')"
```

Comment:
I'd probably change the condition to something like this to not match the `/packages/` page itself:
```suggestion
				v-if="RegExp('^/packages/.+$').test(path)"
```